chain/client/proof.go

```diff
@@ -399,7 +399,8 @@ func MakeOtherStoresMerkleHash(mspo rootmulti.MultiStoreProofOp) (cmn.HexBytes,
 func GetProofHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		vars := mux.Vars(r)
-		requestID, err := strconv.ParseUint(vars[requestIDTag], 10, 64)
+		intRequestID, err := strconv.ParseUint(vars[requestIDTag], 10, 64)
+		requestID := zoracle.RequestID(intRequestID)
```

Comment:
Check for errors first before proceeding. Make it a habit. Same for all places below.